intranet/templates/events/home.html

```diff
@@ -73,7 +73,7 @@ <h2>Events</h2>
                 </a>
```

Comment:
Another thing that I've noticed is that unapproved events don't show up on the top. I'd like them to, so that only approved events show in the calendar.

Reply:
Unapproved events should have their own dedicated section at the top of the page:
`{% for category in events %}
    {% if category.title == "Awaiting Approval" and category.events %}
        <h2 class="category">{{ category.title }}:</h2>
            {% for event in category.events %}  
                 …
` (lines 125-135)